hash/megadriv.xml

```diff
@@ -26654,10 +26967,12 @@ Notice that these are not working on real hardware due to bugged code with VDP i
 		</part>
 	</software>
 
+    <!-- Known as The Story of Thor in other regions -->
```

Comment:
These are unnecessary, they can be derived from the cloneof field.

Reply:
Have removed those comments